rust/gui-client/src-tauri/src/client/deep_link/linux.rs

```diff
@@ -13,7 +12,7 @@ impl Server {
         Ok(Self {})
     }
 
-    pub(crate) async fn accept(self) -> Result<Secret<SecureUrl>, Error> {
+    pub(crate) async fn accept(self) -> Result<SecretString, Error> {
```

Comment:
We can split this out into a different PR if we want. The `SecureUrl` type was originally only meant for the phoenix-channel so I replaced it with a `SecretString` here because I didn't want to keep the type just for this.